homeassistant/components/amcrest/camera.py

```diff
@@ -365,10 +370,15 @@ def update(self):
                     self._brand = "unknown"
             if self._model is None:
                 resp = self._api.device_type.strip()
+                _LOGGER.debug("Device_type=%s", resp)
                 if resp.startswith("type="):
                     self._model = resp.split("=")[-1]
                 else:
                     self._model = "unknown"
+            if self._serial is None:
+                resp = self._api.serial_number.strip()
+                _LOGGER.debug("serial=%s", resp)
+                self._serial = resp if resp else "unknown"
```

Comment:
```suggestion
                self._serial = resp or "unknown"
```
no need to check it

Reply:
Please see my comment above. The whole setting `unknown` isn't needed at all.
As for the unique_id it is checked against it again to make it `None`.
So; make it `None` from the start than we don't need all these `unknown`'s